rtptransceiver.go

```diff
@@ -80,6 +81,33 @@ func (t *RTPTransceiver) getCodecs() []RTPCodecParameters {
 	return filteredCodecs
 }
 
+// SetHeaderExtensions stores header extension from SDP
+func (t *RTPTransceiver) SetHeaderExtensions(headerExtensions []RTPHeaderExtensionParameter) error {
```

Comment:
Would it be ok if we did [RTPSender.SetParameters](https://www.w3.org/TR/webrtc/#dom-rtcrtpsender-setparameters)

I don't see this exposed on the receiver side. Still looking

Reply:
I went with parallel to `SetCodecPreferences` which is on the transceiver now. I think it could be in the transceiver as sender and receiver on a transceiver should use the same codec/header extensions.